source/common/upstream/upstream_impl.h

```diff
@@ -477,12 +477,19 @@ class ClusterImplBase : public Cluster, protected Logger::Loggable<Logger::Id::u
   virtual void startPreInit() PURE;
 
   /**
-   * Called by every concrete cluster when pre-init is complete. At this point, shared init takes
-   * over and determines if there is an initial health check pass needed, etc.
+   * Called by every concrete cluster when pre-init is complete. At this point, SDS init manager
+   * initializes every registered SDS api target.
```

Comment:
update the comment as:
At this point, shared init starts sds_init_manager_ initialization and determines if there is an initial health check pass needed, etc.